### PR TITLE
Feat/AUT-1192/e2e item asset add content to a block

### PR DESCRIPTION
**Pull request by Silvia-loza**
**End to end testing.**
Related to:  https://oat-sa.atlassian.net/browse/AUT-1192
Depends on https://github.com/oat-sa/tao-core/pull/3230
**Description of changes:**
Creation of E2E tests for adding different types of content to A-block
How to run locally:
Checkout to  https://github.com/oat-sa/extension-tao-itemqti/pull/new/feat/AUT-1192/e2e-item-Asset-add-content-to-A-block 
run tests from tao core extn:
`npm run cy:open` - to run in browser
or `npm run cy:run` - for headless execution
File to test:  `add-content-to-A-block-spec.js`
**All tests should pass:**
image

### CODE REVIEW

File: views/cypress/tests/add-content-to-A-block.spec.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA ;
+ */
+
+import urls from '../utils/urls';
+import selectors from '../utils/selectors';
+
+import { selectUploadLocalAsset } from '../utils/resource-manager';
+import { addAblock } from '../utils/authoring-add-interactions';
+import { editText } from '../utils/edit-text-Ablock';
+
```
Comment (adriloma) [inline views/cypress/tests/add-content-to-A-block.spec.js:25]: why those breaklines between imports?

File: views/cypress/utils/authoring-add-interactions.js [hunk 2]
```diff
@@ -74,13 +74,30 @@ export function addBlockAndInlineInteractions() {
     }
 }
 
+/**
+ * Adds A block without inner inline interactions
+ */
+export function addAblock() {
+
```
Comment (adriloma) [inline views/cypress/utils/authoring-add-interactions.js:81]: same here

File: views/cypress/utils/edit-text-Ablock.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 Open Assessment Technologies SA ;
+ */
+const aBlockParagraph = '.widget-box[data-qti-class="_container"] p'
+
+export function  editText() {
+
```
Comment (adriloma) [inline views/cypress/utils/edit-text-Ablock.js:21]: same

**Review by adriloma — APPROVED** [review]
All good, although there are some unexpected breaklines.

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following conventional commits
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by bziondik — APPROVED** [review]
- [x]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**